common/services/catalogue/urls.js

```diff
@@ -160,3 +197,54 @@ export function downloadUrl({
     },
   };
 }
+
+export type CatalogueQuery = {|
+  query: string,
+  page: number,
+  workType: ?(string[]),
+  itemsLocationsLocationType: ?(string[]),
+  _queryType: ?string,
+  _dateFrom: ?string,
+  _dateTo: ?string,
+  _isFilteringBySubcategory: ?string,
+|};
+
+const defaultState: CatalogueQuery = {
+  query: '',
+  page: 1,
+  workType: null,
+  itemsLocationsLocationType: null,
+  _queryType: null,
+  _dateFrom: null,
+  _dateTo: null,
+  _isFilteringBySubcategory: null,
+};
+
+export function searchQueryParams() {
```

Comment:
This is a really smart and neat solution to the hidden params, but maybe worth leaving a comment as it might not be completely obvious to future selves.